.github/workflows/ci.yml

```diff
@@ -424,9 +424,8 @@ jobs:
       - name: Install rust toolchain
         id: toolchain
         run: |
-          rustup toolchain install ${{ needs.env.outputs.msrv }} --profile minimal
-          rustup override set ${{ needs.env.outputs.msrv }}
-
+          rustup toolchain install stable --profile minimal
+          rustup override set stable
```

Comment:
why this change?

Reply:
I had to bump the MSRV for the `s2n-quic-xdp` crate higher than our root MSRV. But I wanted to avoid special casing this in the `crates` task so just using `stable` seemed better, especially since we've already got a specific `test` task to test the MSRV.